pandas/core/frame.py

```diff
@@ -8140,6 +8140,10 @@ def corr(self, method="pearson", min_periods=1) -> "DataFrame":
             to have a valid result. Currently only available for Pearson
             and Spearman correlation.
 
+        axis : {0 or 'index', 1 or 'columns'}, default 0
+            The axis to use. 0 or 'index' to compute column-wise, 1 or 'columns' for
+            row-wise.
```

Comment:
Can you add a version added directive here? There's an example under ddof for the DataFrame.cov method.